docs/codeql/codeql-cli/creating-codeql-query-suites.rst

```diff
@@ -131,8 +133,36 @@ In addition to metadata tags, the keys in the constraint block can also be:
 - ``tags contain all``---each of the given match strings must match one of the
   components of the ``@tags`` metadata property.
 
-Examples
-~~~~~~~~
+Filtering Examples
+~~~~~~~~~~~~~~~~~~
+
+A common use case is to create a query suite that runs all queries in a CodeQL pack,
+except for a few that are known to be problematic. The following three query suite
```

Comment:
```suggestion
except for a few specific queries that the user does not want to run. The following three query suite
```

Reply:
Specify that to do this, we recommend filtering on the query ID. Mention where to find the query ID in the metadata.